devel/commitizen/Portfile

```diff
@@ -43,14 +43,17 @@ depends_run-append \
     port:py${python.version}-tomlkit \
     port:py${python.version}-yaml
 
-default_variants    +bash
+default_variants    +bash_completion
 
-variant bash description {Include Bash completion support} {
-    depends_run-append \
-        port:py${python.version}-argcomplete \
-        port:bash
+variant bash_completion description {Include Bash completion support} {
+    depends_build-append    port:py${python.version}-argcomplete
```

Comment:
to me it seems that this is also needed at [runtime](https://github.com/commitizen-tools/commitizen/blob/5a764bf172d6431e6192678a58dd2b8f7ab8daf6/commitizen/cli.py#L6), so changing it to `depends_build` isn't correct

Reply:
@reneeotten Thank, you're right. argcomplete is needed at runtime.